### ISSUE TITLE
Automatic Updating

### ISSUE THREAD

**Issue by Skyman12** [Skyman12/SteamSaleWatcher#4]
The server should automatically run the update method once per day to get the new sale data. This can take up to 1 hour to finish running, so this may need to run on a separate thread. Once this is complete, the .json file for the games will be updated. 

The update method in the server is already done, this just needs to be automated to once per day.
